pandas/tests/sparse/test_frame.py

```diff
@@ -1132,7 +1132,7 @@ def test_isnotnull(self):
 @pytest.mark.parametrize('dtype', [object, bool, int, float, np.uint16])
```

Comment:
For what I quickly tested (0.18.1) only simple methods and properties, like `.tocoo()`, `todok()`, `.dtype`, `.shape`, work with spmatrices of object dtype. While we do preserve this dtype if encountered, I see little harm in skipping it.